src/utils/helpers.py

```diff
@@ -2,9 +2,67 @@
 import logging
 from pathlib import Path
 
+import yaml
+from pyspark.sql import DataFrame
+from pyspark.sql.types import DecimalType, StringType, TimestampType
+
 PATH_SPARK_LOG = "/tmp/logs/spark.log"
 
 
+def convert_datetime_columns(
+    df: DataFrame, ignore_cols: list[str]
+) -> DataFrame:
+    """
+    iasWorld stores datetimes via the TIMESTAMP data type, but historically
+    we converted these columns to strings to avoid messy type incompatibility
+    with Athena. This function converts TIMESTAMP columns to UTC datetime
+    strings. Columns specified via schema overrides will ignore this conversion.
+
+    Args:
+        df: Spark DataFrame with columns to convert.
+        ignore_cols: Ignore columns specified in this list during conversion.
+
+    Returns:
+        Spark DataFrame with TIMESTAMP columns converted to STRING.
+    """
+    for field in df.schema.fields:
+        if (
+            isinstance(field.dataType, TimestampType)
+            and field.name not in ignore_cols
+        ):
+            df = df.withColumn(field.name, df[field.name].cast(StringType()))
+    return df
+
+
+def convert_decimal_columns(
+    df: DataFrame, ignore_cols: list[str]
+) -> DataFrame:
+    """
+    iasWorld stores many columns as the generic NUMBER Oracle data type, but
+    almost always uses DECIMAL(10,0) for the actual values. This function
+    automatically converts all such columns to DECIMAL(10,0). Columns specified
+    via schema overrides will ignore this conversion.
```

Comment:
Note my warning in the PR body: there's a chance that by doing this we could end up silently truncating numbers in some future where they expand beyond `DECIMAL(10,0)`. 

We _could_ simply leave everything as `DECIMAL(38,10)` to avoid this problem, but the downside is that Athena will then show/return numbers like `1242424.0000000000` even for small integers.

I do not understand why they designed their database this way 🙃